horizons/gui/modules/multiplayermenu.py

```diff
@@ -327,6 +327,7 @@ def show(self):
 		})
 
 		gamenametextfield = self._gui.findChild(name='gamename')
+
```

Comment:
Why not? It separates this code and the next ``def`` keyword.

Reply:
Because the `gamenametextfield` variable only serves as a variable-declaration for the inline function definition. Putting it on an extra line separated from the following code makes it an individual code block to look at, even though it clearly belongs to that function. So imho putting a newline there might adhere to some standard but doesn't help the readability.